.github/workflows/stale-issues-pr.yml

```diff
@@ -12,19 +12,7 @@ jobs:
       issues: write
       pull-requests: write
     steps:
-      # this is a hash for actions/stale@v9.0.0
-      - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e
+      - uses: metamask/github-tools/.github/workflows/stale-issue-pr.yml@ad27b74eec04e404ebf1383429bdcabad227f05e
         with:
-          stale-issue-label: 'stale'
-          only-issue-labels: 'type-bug'
-          exempt-issue-labels: 'type-security'
-          days-before-issue-stale: 90
-          stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity in the last 90 days. It will be closed in 45 days if there is no further activity. The MetaMask team intends on reviewing this issue before close, and removing the stale label if it is still a bug. We welcome new comments on this issue. We do not intend on closing issues if they report bugs that are still reproducible. Thank you for your contributions.'
-          days-before-issue-close: 45
-          close-issue-message: 'This issue was closed because there has been no follow up activity in the last 45 days. If you feel this was closed in error, please reopen and provide evidence on the latest release of the extension. Thank you for your contributions.'
-          stale-pr-label: 'stale'
-          days-before-pr-stale: 60
-          stale-pr-message: 'This PR has been automatically marked as stale because it has not had recent activity in the last 60 days. It will be closed in 14 days. Thank you for your contributions.'
-          days-before-pr-close: 14
-          close-pr-message: 'This PR was closed because there has been no follow up activity in the last 14 days. Thank you for your contributions.'
-          operations-per-run: 600
+          base_ref: main
```

Comment:
I'm not following what's going on here.

The linked workflow doesn't seem to be a reusable workflow, so I'm surprised actionlint didn't throw an error.

And even if it was a reusable workflow, it doesn't accept these parameters